guide/results-viewer-react/src/components/TestResults/model.ts

```diff
@@ -12,7 +12,7 @@ function isObject (o: unknown): o is object {
  * @param check {CheckType | undefined} Type of check (string or function) to use on this key and value.
  * @param unknownCheck {CheckType | undefined} Check to run (if any) if this is not a known key
  * @param key {string} key from parent object
- * @param check {any} value from parent object
+ * @param value {any} value from parent object
  * @returns A string if there is a problem with this key/value pair, undefined otherwise
  */
  function valueChecker (check: CheckType | undefined, unknownCheck: CheckType | undefined, key: string, value: any) {
```

Comment:
Looks like the parameter is still named check and hasn't been updated to value?

Reply:
There's both. There's a `check` and a `value`. The JSDocs just listed `check` twice. This was fixing the docs

`valueChecker(check, unknownCheck, key, value)`

Reply:
Oops needed to see one more line up :)